### PR TITLE
Ont styling

### PR DESCRIPTION
**Pull request by bondarik1995**

### CODE REVIEW

File: app/communities-configs/ont/index.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import coinsBlueIcon from 'images/coinsBlue.svg?inline';
+import OntLogo from 'images/communities-logos/ont.svg?inline';
+
+import RobotoRegularEOT from 'fonts/Roboto/Roboto-Regular.eot';
```
Comment (sergeyi23) [inline app/communities-configs/ont/index.js:5]: Let's move font files to ont subfolder

File: app/components/UserNavigation/index.jsx [hunk 2]
```diff
@@ -171,7 +172,7 @@ const UserNavigation = ({
             id={`redireact-to-edit-${userId}-user-page-2`}
             data-user={userId}
           >
-            <img src={pencilIcon} alt="icon" />
+            <Icon icon={pencilIcon} width="18" />
```
Comment (sergeyi23) [inline app/components/UserNavigation/index.jsx:175]: I see width=18 repeating several times. Maybe create a component for icon of this width?

Comment (sergeyi23) [reply app/components/UserNavigation/index.jsx:175]: Or maybe even several components for icons of different sizes.

File: app/communities-configs/ont/index.js (hunk 1 above)
Comment (bondarik1995) [reply app/communities-configs/ont/index.js:5]: done

**Pull request merged**